common/src/main/java/com/amazonaws/services/schemaregistry/common/AWSSchemaRegistryClient.java

```diff
@@ -90,18 +90,19 @@ public AWSSchemaRegistryClient(@NonNull AwsCredentialsProvider credentialsProvid
                 .retryPolicy(retryPolicy)
                 .addExecutionInterceptor(new UserAgentRequestInterceptor())
                 .build();
-        UrlConnectionHttpClient.Builder urlConnectionHttpClientBuilder = UrlConnectionHttpClient.builder();
-        if (glueSchemaRegistryConfiguration.getProxyUrl() != null) {
-        	log.debug("Creating http client using proxy {}", glueSchemaRegistryConfiguration.getProxyUrl().toString());
-    		ProxyConfiguration proxy = ProxyConfiguration.builder().endpoint(glueSchemaRegistryConfiguration.getProxyUrl()).build();
-    		urlConnectionHttpClientBuilder.proxyConfiguration(proxy);
-        }
+//        UrlConnectionHttpClient.Builder urlConnectionHttpClientBuilder = UrlConnectionHttpClient.builder();
```

Comment:
Please do not include commented out code in Pull Request. Delete them if they should not be included.

Reply:
Why do we need to remove our http client builder in this case?